blivet/osinstall.py

```diff
@@ -1188,6 +1184,165 @@ def write(self):
         Blivet.write(self)
 
         self.make_mtab()
+        self.fsset.write()
+
+    def mountpoints(self):
+        return self.fsset.mountpoints
```

Comment:
The first definition was in FSSet(), this is in InstallerStorage. I wasn't sure if I should leave the property defined in FSSet, even though it seemed a bit redundant.